src/fiatExchanges/SelectProvider.tsx

```diff
@@ -235,13 +236,13 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
         digitalAsset={digitalAsset}
         flow={flow}
       />
-      {coinbaseProvider && coinbasePayVisible ? (
+      {coinbaseProvider && coinbasePayVisible && (
```

Comment:
```suggestion
      {coinbasePayVisible && (
```
coinbasePayVisible is only truthy when coinbaseProvider is also truthy

Reply:
oh just saw your comment that typescript was complaining about this. Very odd that its not understanding